src/frequenz/dispatch/_managing_actor.py

```diff
@@ -159,22 +208,40 @@ async def _handle_dispatch(self, dispatch: Dispatch) -> None:
             dispatch: The dispatch to handle.
         """
         if dispatch.type != self._dispatch_type:
-            _logger.debug("Ignoring dispatch %s", dispatch.id)
+            _logger.debug(
```

Comment:
We don't need this check anymore as the new running_state sender supports filtering by type